openmdao/utils/om.py

```diff
@@ -285,11 +287,16 @@ def _view_metamodel(prob):
 
         pathname = options.pathname
         port_number = options.port_number
+        try:
+            resolution = int(options.resolution)
```

Comment:
when you do the `parser.add_argument` above for resolution, you can set the 'type' arg to int and argparse will automatically type check the value for you.